includes/Modules/Search_Console.php

```diff
@@ -246,6 +246,20 @@ protected function create_data_request( Data_Request $data ) {
 		return new WP_Error( 'invalid_datapoint', __( 'Invalid datapoint.', 'google-site-kit' ) );
 	}
 
+	/**
+	 * Returns a punycode version of a unicode URL.
+	 *
+	 * @param string $url The URL to decode.
+	 */
+	protected function decode_unicode_url( $url ) {
```

Comment:
Potentially call this `unicode_to_punycode`? That may not be a great name in itself either, but it clarifies more what it does.